pkg/ffuf/request_test.go

```diff
@@ -0,0 +1,246 @@
+package ffuf
```

Comment:
I love the added tests <3

Reply:
do they.... pass the test?

![image](https://user-images.githubusercontent.com/5291556/151623635-6cbcfad4-9dff-44f1-8279-34b152b7af15.png)